docs/backend/openai_api_completions.ipynb

```diff
@@ -221,7 +221,14 @@
    "metadata": {},
    "source": [
     "## Structured decoding (JSON, Regex)\n",
-    "You can specify a JSON schema or a regular expression to constrain the model output. The model output will be guaranteed to follow the given constraints.\n",
+    "You can define a JSON schema or regular expression to constrain the model's output, which depends on the grammar backend.\n",
```

Comment:
You can define a JSON schema or regular expression to constrain the model's output. The model output will be guaranteed to follow the given constraints and this depends on the grammar backend.

Reply:
done, thanks